mne/viz/utils.py

```diff
@@ -2441,3 +2495,24 @@ def _check_cov(noise_cov, info):
              'scaling of magnetometers and gradiometers when viewing data '
              'whitened by a noise covariance')
     return noise_cov
+
+
+def _get_channel_types(info, picks=None, unique=True,
```

Comment:
see if _picks_by_type in pick.py or _get_ch_type in channels.py does not already do what you want. Such a function should not be in viz functions.